Uplift/Views/Onboarding/CheckBoxView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  CheckBoxView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct CheckBoxView: View {
+
+    // MARK: - Properties
+
+    @Binding var didCheckTerms: Bool
+    @Binding var didCheckData: Bool
+    @Binding var didCheckLocation: Bool
```

Comment:
```suggestion
    @Binding var didCheckData: Bool
    @Binding var didCheckLocation: Bool
    @Binding var didCheckTerms: Bool
```
Fix for alphabetical order.